docs/Node/ReadLine.md

```diff
@@ -50,33 +66,33 @@ Prompt the user for input on the specified `Interface`.
 #### `setPrompt`
 
 ``` purescript
-setPrompt :: forall eff. String -> Int -> Interface -> Eff (console :: CONSOLE | eff) Interface
+setPrompt :: forall eff. String -> Int -> Interface -> Eff eff Interface
```

Comment:
I think `Completer` is probably ok, because `noCompleter` does intuitively seem to have the right type, as far as I can tell. I also think it's probably safe to assume that the row type argument for `Interface` is not an empty row, because all streams should have a nonempty effect row too &mdash; if you somehow get a stream which has an empty effect row, that's effectively saying that calling stream functions like `onData` or `end` on it have no effects. So any module that gives you access to some stream should take responsibility for making sure the effect row is not empty.

btw @paf31 - if the above is correct, we should probably document that in `node-streams`?